core/server/worker/src/main/java/alluxio/worker/netty/NettyDataServer.java

```diff
@@ -54,14 +60,21 @@ public final class NettyDataServer implements DataServer {
   /**
    * Creates a new instance of {@link NettyDataServer}.
    *
-   * @param address the server address
-   * @param workerProcess the Alluxio worker process
+   * @param nettyBindAddress the server address
+   * @param ufsManager       the UfsManager object
+   * @param doraWorker       the DoraWorker object
    */
-  public NettyDataServer(final SocketAddress address, final WorkerProcess workerProcess) {
-    mSocketAddress = address;
-    mBootstrap = createBootstrap().childHandler(new PipelineHandler(workerProcess));
+  @Inject
+  public NettyDataServer(
+      @Named("NettyBindAddress") InetSocketAddress nettyBindAddress,
+      UfsManager ufsManager,
+      @Nullable DoraWorker doraWorker) {
```

Comment:
why dora worker is nullable? will it be a problem for this class?

Reply:
It's a mistake. I removed the annotation `@Nullable`.